test/geometries/polyhedral_surface.cpp

```diff
@@ -0,0 +1,86 @@
+// Boost.Geometry
+// Unit Test
+
+// Copyright (c) 2025 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+// Polyhedral surface tests
+
+#include <iostream>
+#include <initializer_list>
+
+#include <geometry_test_common.hpp>
+
+#include <boost/geometry/geometry.hpp>
+#include <boost/geometry/geometries/point.hpp>
+#include <boost/geometry/geometries/polygon.hpp>
+#include <boost/geometry/geometries/polyhedral_surface.hpp>
+
+int test_main(int, char* [])
+{
+    namespace bg = boost::geometry;
+    using point_t = bg::model::point<double, 3, bg::cs::cartesian>;
+    using polygon_t = bg::model::polygon<point_t>;
+    using polyhedral_t = bg::model::polyhedral_surface<polygon_t>;
+    using const_polyhedral_t = bg::model::polyhedral_surface<polygon_t> const;
+
+    // intializing an empty polyhedral surface (default constructor)
+    polyhedral_t polyhedral0;
+    BOOST_CHECK_EQUAL(boost::size(polyhedral0), 0);
+
+    // creating a polyhderal surface using standard initiallized list
+    polyhedral_t polyhedral1 = {{{{0, 0, 0}, {0, 1, 0}, {1, 1, 0}, {1, 0, 0}, {0, 0, 0}}},\
+                                {{{0, 0, 0}, {0, 1, 0}, {0, 1, 1}, {0, 0, 1}, {0, 0, 0}}},\
+                                {{{0, 0, 0}, {1, 0, 0}, {1, 0, 1}, {0, 0, 1}, {0, 0, 0}}},\
+                                {{{1, 1, 1}, {1, 0, 1}, {0, 0, 1}, {0, 1, 1}, {1, 1, 1}}},\
+                                {{{1, 1, 1}, {1, 0, 1}, {1, 0, 0}, {1, 1, 0}, {1, 1, 1}}},\
+                                {{{1, 1, 1}, {1, 1, 0}, {0, 1, 0}, {0, 1, 1}, {1, 1, 1}}}};
```

Comment:
Same comments as for the example.
I think this unit test is fine, testing 3 use cases.
But the example does not need to be an exact repetition and one example is enough.

Reply:
Thanks, I removed the `example/08_polyhedralsurface_example.cpp` and kept the one in the docs.